build.gradle

```diff
@@ -60,6 +60,7 @@ buildscript {
                 core              : '1.3.2',
                 databinding       : deps.gradleAndroidPlugin,
                 fragment          : '1.3.1',
+                hilt              : '1.0.0-beta01',
```

Comment:
If I remember correctly, ```hilt``` helps to remove/generate the code from ```Dagger``` but does the same thing(dependency injection)?

Reply:
Hilt is part of Dagger, this hilt version deals with the AndroidX Hilt extensions, which provide the `HiltWorkerFactory`